config/default.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
   theme: 'dark',
   currency: 'USD',
   locale: 'en',
+  clockFormat24: false,
```

Comment:
I’d be more included to make this something like `timeDisplayMode` and make it an enum with values like `24hour` & `12Hour` and then make the widget on the settings page a dropdown so user can select the one they want. It’s a bit more explicit so that the user knows what they get if they don’t select 24 hour clock, and leaves the option open to introduce other time formats.